build/git_revision_test.go

```diff
@@ -25,7 +24,15 @@ var (
 func TestGitRevision_terraform(t *testing.T) {
 	testutil.EndToEndTest(t)
 
-	gr := &GitRevision{Product: product.Terraform}
+	tempDir, err := os.MkdirTemp("", "license")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
```suggestion
	tempDir := t.TempDir()
```

This is more or less equivalent functionality except that upstream also takes care of removing the directory after test runs. See https://pkg.go.dev/testing#T.TempDir